src/nix-env/nix-env.cc

```diff
@@ -919,6 +919,14 @@ static void queryJSON(Globals & globals, vector<DrvInfo> & elems)
         pkgObj.attr("version", drvName.version);
         pkgObj.attr("system", i.querySystem());
 
+        if (printOutPath) {
+            DrvInfo::Outputs outputs = i.queryOutputs();
+            JSONObject outputObj = pkgObj.object("outputs");
+            for (auto & j : outputs) {
+                outputObj.attr(j.first, j.second);
+            }
+        }
+
         JSONObject metaObj = pkgObj.object("meta");
```

Comment:
According to the `nix-env --help` this should only be done if the `--meta` command line is given.
```

       • --meta
         Print all of the meta-attributes of the derivation. This option is only available with  --xml  or
         --json.
```

@edolstra should I file a bug for this and fix?

Reply:
Yes, please do.